doc/deoplete.txt

```diff
@@ -429,6 +429,8 @@ member						*deoplete-source-member*
 tag						*deoplete-source-tag*
 		It collects keywords from |ctags| files.
 		Note: It only supports UTF-8 encoding tag file.
+		Note: https://github.com/Shougo/neoinclude.vim is required for
+		this.
```

Comment:
Oh.. but I was not seeing tag sources (at some time), and when digging into the source have seen the reference to neoinclude.
Need to change that / investigate then.